advanced/am-pattern-3/templates/am/wso2am-pattern-3-conf-entrypoint.yaml

```diff
@@ -0,0 +1,88 @@
+# Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
@Chathuru I don't think we need to override the entry point script (except in the case of Publisher and DevPortal where solr indexing is used), IIRC. Profile optimization supported within the image itself. If the images are not available in Docker Hub, we need to push them to the Docker Hub.

Reply:
Fix https://github.com/wso2/kubernetes-apim/pull/365/commits/3588e9f5044c3947741ef019ecda0ee150daad18